src/context/WalletProvider/KeepKey/components/Pin.tsx

```diff
@@ -25,6 +26,7 @@ const moduleLogger = logger.child({ namespace: ['Pin'] })
 export const KeepKeyPin = () => {
   const [error, setError] = useState<string | null>(null)
   const [loading, setLoading] = useState(false)
+  const [isEmpty, setIsEmpty] = useState(true)
```

Comment:
Nitpick but by reading the code without reverse engineering, I don't know what is empty there

I would probably call it `isPinEmpty` or `isInputEmpty` and same for the setter